src/Sql/Sql/VulnerabilityAssessment/Cmdlet/VulnerabilityAssessmentSettings/ManagedDatabase/UpdateAzureSqlManagedDatabaseVulnerabilityAssessmentSettings.cs

```diff
@@ -68,5 +68,20 @@ protected override object TransformModelToOutputObject(VulnerabilityAssessmentSe
         {
             return new ManagedDatabaseVulnerabilityAssessmentSettingsModel(model, GetServerName(), GetDatabaseName());
         }
+
+        /// <summary>
+        /// Overriding to add warning message in case cmdlet is on master
+        /// </summary>
+        public override void ExecuteCmdlet()
+        {
+            var database = GetDatabaseName();
+            if (database == "master" || database == "msdb" || database == "model")
```

Comment:
This is repeated many times. It would make sense to make a common ancestor for ManagedInstance cmdlets